instant-xml-macros/src/lib.rs

```diff
@@ -40,6 +44,12 @@ impl ContainerMeta {
                 MetaItem::Attribute => panic!("attribute key invalid in container xml attribute"),
                 MetaItem::Ns(ns) => meta.ns = ns,
                 MetaItem::Rename(lit) => meta.rename = Some(lit),
+                MetaItem::RenameAll(lit) => {
+                    meta.rename_all = match RenameRule::from_str(&lit.to_string()) {
+                        Ok(rule) => rule,
+                        Err(err) => panic!("{err}"),
```

Comment:
This should use a `syn::Error`, and we should use the `lit` so that we can preserve the span. (As such, I don't think we need the `ParseError` to carry the unknown name?)